POC: Increase utilities test coverage (#224)
BBC-archive/psammead: packages/utilities/psammead-test-helpers/README.md

| shallowRender              | component        |  Shallow renders the component using react-test-renderer and returns the render output |
| shouldShallowMatchSnapshot | title, component | Shallow renders the component using react-test-renderer and asserts that it matches the given snapshot, which will be saved in the `__snapshots__` directory. The first argument `title` is the title for the test. |
| isNull                     | title, component    | Renders the component using react-test-renderer, converts it to JSON and asserts that it is null. The first argument `title` is the title for the test. |

> | testUtilityPackages                     | actualExports, expectedExports, utilityName | Compares an imported utility package and compares the exported values against a keyed object which validates that all the utilities exports are explicit. |

Reviewer: ```suggestion
| testUtilityPackages        | actualExports, expectedExports, utilityName | Validates an imported utility package's exported values against an object of key-value pairs in the form `{ name_of_export: 'type of export' }`, e.g. `{ shouldMatchSnapshot: 'function' }`. |
```

Reviewer: (really just wanted to fix up the spacing to line up with the `|` above it - but had a tinker with the wording while I was there! 😇 )

## Contributing

> When **adding** a new export to this utility package the [export tests](https://github.com/BBC-News/psammead/blob/5d7395fd60bd8d73796d5a23775b4b5b36db1445/packages/utilities/psammead-test-helpers/index.test.jsx#L8-L14) also need to be updated and the export should be adding to the [README](https://github.com/BBC-News/psammead/tree/5d7395fd60bd8d73796d5a23775b4b5b36db1445/packages/utilities/psammead-test-helpers#exported-functions). When **removing** an exisiting export from this utility package the [export tests](https://github.com/BBC-News/psammead/blob/5d7395fd60bd8d73796d5a23775b4b5b36db1445/packages/utilities/psammead-test-helpers/index.test.jsx#L8-L14) need to be updated, the export should be removed from the [README](https://github.com/BBC-News/psammead/tree/5d7395fd60bd8d73796d5a23775b4b5b36db1445/packages/utilities/psammead-test-helpers#exported-functions) and the package version requires a major change (EG: 1.2.1 -> 2.0.0) as this would be considered a breaking change due to functionality being removed.

Reviewer: ```suggestion
When **adding** a new export to this utility package, the export must be added to the [export tests](https://github.com/BBC-News/psammead/blob/5d7395fd60bd8d73796d5a23775b4b5b36db1445/packages/utilities/psammead-test-helpers/index.test.jsx#L8-L14). When **removing** an existing export from this utility package, the export will need removing from the export tests and the package version requires a major change (EG: 1.2.1 -> 2.0.0). In all cases, the README requires updating too.
```
